src/detect-bsize.c

```diff
@@ -266,6 +268,43 @@ static DetectBsizeData *DetectBsizeParse (const char *str)
     return bsz;
 }
 
+static bool DetectBsizeCheckContent(Signature *s, int list, DetectBsizeData *bsz)
+{
+    SigMatch *sm = s->init_data->smlists[list];
+    for (; sm != NULL; sm = sm->next) {
+        if (sm->type != DETECT_CONTENT)
```

Comment:
while at it, it might be useful to see if we can find other conflicts too? E.g. on the uri buffer we can have `urilen`, which does pretty much the same as `bsize`. Would be good they don't have conflicting, mutually exclusive settings.

Reply:
Should we address  `urilen` potential conflicts in another redmine issue?